### PR TITLE
Added a share button

### PR DESCRIPTION
**Pull request by OmarJalil**
Added a share button to improve UX, because many users don't know they can longpress.

### CODE REVIEW

File: Sources/Classes/ViewController.swift [hunk 1]
```diff
@@ -298,10 +323,30 @@ fileprivate extension ViewController {
...
+                shareButton.leftAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leftAnchor, constant: 0.0),
+                shareButton.heightAnchor.constraint(equalToConstant: 52.0),
+                shareButton.widthAnchor.constraint(equalToConstant: 52.0),
+                ].forEach { $0.isActive = true }
+        } else {
+            [
+                shareButton.topAnchor.constraint(equalTo: view.topAnchor, constant: 0.0),
+                shareButton.leftAnchor.constraint(equalTo: view.leftAnchor, constant: 0.0),
+                shareButton.heightAnchor.constraint(equalToConstant: 52.0),
+                shareButton.widthAnchor.constraint(equalToConstant: 52.0),
```
[the remark this answers is not in the document]
Comment (OmarJalil) [reply Sources/Classes/ViewController.swift:344]: I copied and pasted the code from above, I just don't see the problem

**Comment by OmarJalil** [comment]
@nakajijapan Is the Travis error something I did wrong? If that so, what can I do to fix it?

File: README.md [hunk 2]
```diff
@@ -181,6 +181,10 @@ class PhotoSliderSDImageLoader: PhotoSlider.ImageLoader {
         progress: @escaping PhotoSlider.ImageLoader.ProgressBlock,
         completion: @escaping PhotoSlider.ImageLoader.CompletionBlock)
     {
+        //Webp compatibility (optional)
```
Comment (nakajijapan) [inline README.md]:
```suggestion
        // Webp compatibility (optional)
```

File: Sources/Classes/ImageView.swift [hunk 3]
```diff
@@ -11,6 +11,7 @@ import Kingfisher
 @objc protocol PhotoSliderImageViewDelegate {
     func photoSliderImageViewDidEndZooming(_ imageView: PhotoSlider.ImageView, atScale scale: CGFloat)
     func photoSliderImageViewDidLongPress(_ imageView: PhotoSlider.ImageView)
+    func photoSliderImageViewDidSingleTap()
```
Comment (nakajijapan) [inline Sources/Classes/ImageView.swift:14]:
```suggestion
    func photoSliderImageViewDidSingleTap(_ imageView: PhotoSlider.ImageView)
```

File: Sources/Classes/ImageView.swift [hunk 4]
```diff
@@ -207,6 +211,10 @@ extension ImageView {
         delegate?.photoSliderImageViewDidLongPress(self)
     }
 
+    @objc func didSingleTap(_ sender: UIGestureRecognizer) {
+        delegate?.photoSliderImageViewDidSingleTap()
```
Comment (nakajijapan) [inline Sources/Classes/ImageView.swift:215]:
```suggestion
        delegate?.photoSliderImageViewDidSingleTap(self)
```

File: Sources/Classes/ImageView.swift (hunk 3 above)
Comment (nakajijapan) [reply Sources/Classes/ImageView.swift:14]: I want to have consistency, so could you add self to this arguments.

Comment (nakajijapan) [reply Sources/Classes/ImageView.swift:14]: Maybe fixed if you tap "resolve conversation" button.

**Comment by nakajijapan** [comment]
@OmarJalil Thank you for your PR.

I think status becomes stable if you pull the master.

**Pull request merged**